.pylintrc

```diff
@@ -62,10 +62,12 @@ max-line-length=80
 indent-string='    '
 
 allowed-comment-prefixes=int,str,float,bool,v
+# TODO(#12920): Add import a subset of platform i.e the models to forbidden imports
```

Comment:
I don't actually understand what this TODO is saying, and the linked issue doesn't really give much more context. Could you please try to phrase this more clearly and add more details to the issue?